client/blocks/reader-full-post/_content.scss

```diff
@@ -81,27 +81,15 @@
 	.image-wrapper {
 		position: relative;
 
-		.image-border {
-			content: "";
-			position: absolute;
-			pointer-events: none;
-			top: 0;
-			left: 0;
-			bottom: 0;
-			right: 0;
-			box-shadow: inset 0 0 0 1px rgba(0, 0, 0, 0.1);
-			border-radius: 6px; /* stylelint-disable-line scales/radii */
-			margin: auto;
-			max-width: 100%;
-		}
-
 		figcaption {
 			border-bottom-left-radius: 6px; /* stylelint-disable-line scales/radii */
 			border-bottom-right-radius: 6px; /* stylelint-disable-line scales/radii */
 		}
 	}
 	.image-wrapper img {
+		border-radius: 6px; /* stylelint-disable-line scales/radii */
 		display: inherit;
+		border: 1px solid #E5E5E5;
```

Comment:
This is the effective color of the border when it was implemented as a `box-shadow`.

Reply:
Can we use a variable from `color-studio`?

Reply:
Done 👍